README.md

```diff
@@ -108,6 +108,10 @@ Minimum requirements -
 4. Python 3.6.7+
 5. CMake (>= 3.10.2)
 
+Optional requirements -
+
+1. autoconf (required to build optional htslib package)
```

Comment:
can this be changed to `required to output SAM/BAM files`?

Reply:
fixed